config_test.go

```diff
@@ -597,6 +606,53 @@ func Test_compiledModule_Name(t *testing.T) {
 	}
 }
 
+func Test_compiledModule_CustomSections(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    *compiledModule
+		expected []string
+	}{
+		{
+			name:     "no custom section",
+			input:    &compiledModule{module: &wasm.Module{}},
+			expected: nil,
+		},
+		{
+			name: "name",
+			input: &compiledModule{module: &wasm.Module{
+				CustomSections: []*wasm.CustomSection{
```

Comment:
[`name`](https://webassembly.github.io/spec/core/appendix/custom.html#name-section) and [`dwarf_**` (DWARF)](https://yurydelendik.github.io/webassembly-dwarf/) custom sections are handled differently and not stored in wasm.Module.CustomSections, though not sure the users want to access them directly. are we ok with it?

Reply:
cc @codefromthecrypt I think we are fine as long as document this behavior on the API like "DWARF and name sections are not retrieved" since I think no one wants them to access, though this seems weird asymmetry in the behavior.

Reply:
I feel like we can skip these sections and wait for someone to ask for them. if that happens we'd retain the raw bytes of them

Reply:
cool good

Reply:
resolving